src/jdk.attach/linux/classes/sun/tools/attach/VirtualMachineImpl.java

```diff
@@ -343,7 +339,7 @@ private int getNamespacePid(int pid) throws AttachNotSupportedException, IOExcep
         Path statusPath = Paths.get(statusFile);
 
         try {
-            for (String line : Files.readAllLines(statusPath, StandardCharsets.UTF_8)) {
+            for (String line : Files.readAllLines(statusPath, UTF_8)) {
```

Comment:
The 1-arg readAllLines is specified to use UTF-8 so you can drop the second parameter here if you want.

Reply:
Thank you for the suggestion, I have fixed this and a few other places as well.